packages/legend-studio/src/components/editor/edit-panel/service-editor/testable/ServiceTestDataEditor.tsx

```diff
@@ -377,7 +377,7 @@ export const ServiceTestDataEditor = observer(
               <div className="binding-editor__header">
                 <div className="binding-editor__header__title">
                   <div className="binding-editor__header__title__label">
-                    connections
+                    connectionsTestData
```

Comment:
Since this is the title badge, we want to present it a little more user-friendly.

```suggestion
                    connections test data
```